README.md

```diff
@@ -1,147 +1,20 @@
-# Xamarin Component for Google Play Services Client Library
+# Google Play Services / Firebase for Xamarin.Android
 
-Xamarin creates and maintains Xamarin.Android bindings for the Google Play Services Client Library, including:
+Xamarin creates and maintains Xamarin.Android bindings for Google Play Services and Firebase.
 
- - Ads
- - Analytics
- - AppIndexing
- - AppInvite
- - AppState
- - Auth
- - Base
- - Basement
- - Cast
- - Drive
- - Fitness
- - Games
- - GCM
- - Identity
- - Location
- - Maps
- - Measurement
- - Nearby
- - Panorama
- - Plus
- - SafetyNet
- - Support Wearable
- - Vision
- - Wallet
- - Wearable
+## What is Google Play Services / Firebase
 
+Google Play Services and Firebase (GPS/FB) are a set of libraries that allow Android apps to take advantage of Google APIs and services.

```

Comment:
to be precise: 

Google Play Services, Firebase and ML Kit (GPS/FB/MLKit)

Reply:
Good call!